ide/che-ide-gwt-app/src/main/resources/org/eclipse/che/ide/public/loader.js

```diff
@@ -17,6 +17,11 @@ class KeycloakLoader {
         const msg = "Cannot load keycloak settings. This is normal for single-user mode.";
 
         return new Promise((resolve, reject) => {
+            if (window.parent && window.parent['_keycloak']) {
+                window['_keycloak'] = window.parent['_keycloak'];
+                resolve(window.parent['_keycloak']);
```

Comment:
maybe we should resolve to `our` keycloak (window['_keycloak'])

Reply:
yes, probably. In fact I added line 21 after line 22, which explains why it remained like that ;-)
Not sure that makes a real difference though since both point to the same object, and both are set only at page load.

Reply:
fixed